src/components/Sidebar.js

```diff
@@ -11,12 +10,19 @@ export default class Sidebar extends React.Component{
 
   }
 
-  handleToggleSidebar(e) {
+  handleToggleSidebar() {
     this.state.sidebarOpen === "open" ?
       this.setState({sidebarOpen: "close"}) :
       this.setState({sidebarOpen: "open"})
   }
 
+  handleKeyToggleSidebar(e) {
+    console.log({e});
```

Comment:
Not sure if you intended to leave this console.log here

Reply:
whoops, nope.